src/richie/apps/core/templates/richie/error.html

```diff
@@ -0,0 +1,20 @@
+{% extends "richie/base.html" %}
+{% load i18n cms_tags %}
+
+{% block head_title %}
+    {{status}} - {{ title }}
+{% endblock head_title %}
+
+{% block breadcrumbs %}{% endblock breadcrumbs %}
+
+
+{% block content %}
+<div class="error">
+  <svg class="error-icon"><use xlink:href="#icon-search-fail" /></svg>
```

Comment:
Didn't know about this `use` SVG tag. Have you seen this deprecation message? https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href

Reply:
I missed that. Good catch, we have to update all your templates.

Reply:
You'll handle this in another PR?

Reply:
Yes, it's clearer

Reply:
#1234